WordPress/src/main/java/org/wordpress/android/ui/pages/PageListFragment.kt

```diff
@@ -103,4 +106,34 @@ class PageListFragment : Fragment() {
         }
         adapter.update(pages)
     }
+
+    fun scrollToPage(pageId: Long) {
+        val position = viewModel.pages.value?.indexOfFirst { it is Page && it.id == pageId }
+        if (position != null && position != -1) {
```

Comment:
Do we really want to silently fail if the position is invalid? I'd consider either throwing an exception or logging a message. Wdyt?

Reply:
If a page is missing then we don't want to crash the app. It means the page might have been deleted or something. So all that will happen is that the Pages activity is displayed without the scrolling. I think failing silently is ok here.

Reply:
👍